lib/explorer/shared.ex

```diff
@@ -64,11 +64,11 @@ defmodule Explorer.Shared do
 
   def normalise_dtype(dtype) when dtype in @scalar_types, do: dtype
   def normalise_dtype(dtype) when dtype in [:float, :f64], do: {:f, 64}
-  def normalise_dtype(dtype) when dtype in [:integer, :i64], do: {:s, 64}
+  def normalise_dtype(dtype) when dtype in [:integer, :i64, :s64], do: {:s, 64}
```

Comment:
We don't have i8/16/32/64 in Nx, so we shouldn't expose them here. It should be s8/s16/s32/s64 (it seems I missed this in a previous PR, apologies).

Reply:
No problem! I changed it to follow the `s*` naming.